Src/PerformanceCollector/Shared/Implementation/StandardPerformanceCollector/StandardPerformanceCollector.cs

```diff
@@ -294,9 +294,9 @@ private void RegisterPerformanceCounter(string originalString, string reportAs,
                         usesInstanceNamePlaceholder,
```

Comment:
do we really need to add it if we failed to read the counter? How it will be used?